tools/buildutils/cw/Containerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:stable AS base
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update -y && apt upgrade -y
+RUN apt install -y sudo devscripts
+
+FROM base AS with_bazel
+
+RUN --mount=source=.,target=/mnt,type=bind \
+  /mnt/tools/buildutils/installbazel.sh
+
+FROM with_bazel
+
+COPY ./tools/buildutils/build_package.sh /build_package.sh
+
+ENTRYPOINT ["/build_package.sh"]
```

Comment:
IIUC this calls `build_package.sh` without parameters, but that script expects one. Either make the parameter optional (by giving it a default value in the script) or pass a parameter here.

Reply:
The parameter should be passed into the `docker run` command, for example:

```
docker run -v=.:/mnt/build -w /mnt/build android-cuttlefish-build:latest [ARG]

docker run -v=.:/mnt/build -w /mnt/build android-cuttlefish-build:latest base

docker run -v=.:/mnt/build -w /mnt/build android-cuttlefish-build:latest frontend
```

Reply:
Done!